internal/lido/contracts/csmodule/keys_test.go

```diff
@@ -0,0 +1,36 @@
+package csmodule
+
+import (
+	"math/big"
+	"testing"
+)
+
+func TestKeysStatus(t *testing.T) {
```

Comment:
Why testing only for Node Operator ID `-1`? Please improve test coverage here. Also for Signing keys

Reply:
[Done](https://github.com/NethermindEth/sedge/pull/401/commits/d8704a467fe0ee9f41517706e62059f2ffb545c2)

Reply:
I have removed `SigningKeys` function. It is not part of the requirements, having it would be unnecessary.